README.md

```diff
@@ -206,6 +206,44 @@ Example:
 }
 ```
 
+#### `PUT /api/v1/users/:id/events/:event_id`
+Updates a user_event specified by the user ID and event ID. Updates the status of the event. 
```

Comment:
This is great. It's very clear what one needs to do here, thanks for being so detailed.